backend/app.server.js

```diff
@@ -57,21 +57,56 @@ app.get("/", (req, res) => {
   res.render("index");
 });
 
+// routes
+app.get("/new", (req, res) => {
+
+  const newRoom = {
+    "id": req.query.roomId,
+    "name": req.query.roomName,
+    "disableMeeting": false,
+    "temporary":true
+  }
+
+  let found = app.locals.roomsDetail.find(element => element.id == req.query.roomId);
+
+  if(!found){
+      app.locals.roomsDetail.splice( 1, 0, newRoom);
+  }
+
+  res.redirect(`/office#${req.query.roomId}`);
+});
+
+
+app.get("/remove", (req, res) => {
+
+
+    app.locals.roomsDetail = app.locals.roomsDetail.filter(function(value, index, arr){
+
+    return (value.id !=req.query.roomId) && value.temporary!=true;
```

Comment:
```suggestion
    return value.id !== req.query.roomId || value.temporary !== true;
```

Reply:
I believe we have to use operator && then operator ||, because this is a filter and the filter, has to return all other rooms

Reply:
I think filter with `&&` will remove all the temporary rooms too. 😅